src/overlays/actors/ovl_En_Niw/z_en_niw.c

```diff
@@ -138,7 +138,7 @@ void EnNiw_Init(Actor* thisx, PlayState* play) {
     Actor_SetScale(&this->actor, 0.01f);
 
     if (this->niwType == NIW_TYPE_UNK1) {
-        // @Bug this unused variant is broken and crashes on spawn (EnNiw_Update expects a parent, NULL)
+        // @bug this unused variant is broken and crashes on spawn (EnNiw_Update expects a parent, NULL)
         //   if modified to change niwType to TYPE_REGULAR here, new size is smaller than normal
         //   theory: was meant to be a small hand held cucco for grog to show the player
```

Comment:
```suggestion
        //! @bug this unused variant is broken and crashes on spawn (EnNiw_Update expects a parent, NULL)
        //! if modified to change niwType to TYPE_REGULAR here, new size is smaller than normal
        //! theory: was meant to be a small hand held cucco for grog to show the player
```
And could do with rephrasing a bit, the bracket in the first sentence doesn't make much sense. Also capitalisation.

Reply:
So got curious and looked a little bit into this. I don't think it should be a bug statement. Instead a wrote a little note on the type enum mentioning that is meant to be spawned as a child, and expects a non-null parent.

Reply:
Yeah its not a bug, its an unused variant, maybe grog was supposed to hold a chicken to show they had grown up, or there was going to be a pocket chicken like in OOT, but that's all speculation.

Not sure what to mark it, note?

Reply:
I added a comment in the header file. Does that seem fine?